doc/operations/code_upgrade.rst

```diff
@@ -177,7 +177,7 @@ Procedure
 - Members should be use the ``set_constitution`` proposal action to update the constitution scripts.
 - See :ref:`bundle deployment procedure <build_apps/js_app_bundle:Deployment>` to update the JavaScript/TypeScript application.
 
-6. Finally, once the code upgrade process has been successful, the old code version (i.e. the code version run by nodes 0, 1 and 2) can be removed using the ``remove_node_code`` or ``remove_security_policy`` proposal actions.
+6. Finally, once the code upgrade process has been successful, the old code version (i.e. the code version run by nodes 0, 1 and 2) can be removed using the ``remove_node_code`` or ``remove_host_data`` proposal actions.
```

Comment:
remove_snp_host_data I think, we want to namespace this.

Reply:
5143a02dcb85d2aacabb5c1fd88e756aed95166a